lib/unifex/code_generator.ex

```diff
@@ -22,16 +22,32 @@ defmodule Unifex.CodeGenerator do
 
   defp get_generator(%Specs{name: name, interface: nil}) do
     {:ok, bundlex_project} = Bundlex.Project.get()
+    config = bundlex_project.config
 
-    [:nifs, :cnodes]
-    |> Enum.find(&(bundlex_project.config |> Keyword.get(&1, []) |> Keyword.has_key?(name)))
-    |> case do
-      :nifs -> Unifex.CodeGenerators.NIF
-      :cnodes -> Unifex.CodeGenerators.CNode
-    end
+    [:natives, :libs]
+    |> Enum.find(&(config |> Keyword.get(&1, []) |> Keyword.has_key?(name)))
+    |> (fn type ->
+          interfaces = Keyword.get(config[type][name], :interface, []) |> Bunch.listify()
+
+          case interfaces do
+            [] -> Unifex.CodeGenerators.NIF
+            _ -> get_generator(get_generator_module_name(List.first(interfaces)))
+          end
+        end).()
   end
 
   defp get_generator(%Specs{interface: interface}) do
+    get_generator(interface)
+  end
+
+  defp get_generator(interface) do
```

Comment:
This clause accepting bare interface instead of specs is confusing. Can't this just be a part of `get_generator_module_name/1`?

Reply:
done